ddtrace/span.py

```diff
@@ -334,21 +335,26 @@ def _set_str_tag(self, key, value):
         U+FFFD.
         """
         try:
-            self.meta[key] = ensure_text(value, errors="replace")
+            self._meta[key] = ensure_text(value, errors="replace")
         except Exception as e:
             if config._raise:
                 raise e
             log.warning("Failed to set text tag '%s'", key, exc_info=True)
 
     def _remove_tag(self, key):
         # type: (_TagNameType) -> None
-        if key in self.meta:
-            del self.meta[key]
+        if key in self._meta:
+            del self._meta[key]
 
     def get_tag(self, key):
         # type: (_TagNameType) -> Optional[Text]
         """Return the given tag or None if it doesn't exist."""
-        return self.meta.get(key, None)
+        return self._meta.get(key, None)
+
+    def _get_tags(self):
```

Comment:
actually, should this be called _get_str_tags or _get_meta_tags? Seems like we're running into another ambiguous usage of "tags"

Reply:
I'm inclined to leave it as is here until we tackle the problem in reconsidering the overloaded `set_tag` method.